build-ps/build-binary.sh

```diff
@@ -328,23 +328,44 @@ fi
         export CC=${GALERA_CC:-gcc}
         export CXX=${GALERA_CXX:-g++}
 
+        # Look for boost_program_options static library
```

Comment:
Please add a comment why this is needed.

Reply:
added a comment